typescript-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@moonbeam-network/api-augment",
-  "version": "0.3200.0",
+  "version": "0.3300.0",
```

Comment:
I guess we should not commit this change until RT3300 is released, what do you think @pLabarta ?

Reply:
Ah, just commented on the same thing. If we follow our usual process, we should not bump the version till the release, but it has no real impact.